docs/relay-architecture.md

```diff
@@ -71,7 +71,7 @@ Advanced search modes for content discovery:
 
 Example:
 ```json
-{"kinds": [34236], "search": "sort:hot", "limit": 50}
+{"kinds": VIDEO_KINDS, "search": "sort:hot", "limit": 50}
```

Comment:
I don't think we should be removing the other filters here when we switch the kind reference.